migrations/20220620205322-remove-prefixes-from-tags.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+module.exports = {
+  up: async queryInterface => {
+    await queryInterface.sequelize.query(`
+      UPDATE "Collectives" SET tags = array(SELECT regexp_replace(array_to_string(tags,', '), '^#| #| ', '', 'g')) WHERE tags IS NOT NULL AND EXISTS(SELECT * FROM unnest(tags) tag WHERE tag ILIKE '#%');
```

Comment:
Would be great to indent the query in that kind of place, it's difficult to parse.

The approach of `stringify(tags) |> regex_replace(...) |> to_array(...)` chosen here seems dangerous as it's going to split tags that include a comma (see https://opencollective.com/cutiebipocfest: `queer trans* inter* black, indigenous and people of color`). We need to find a more elegant solution, ideally not stringifying the array.

Reply:
I've changed the script to not use `array_to_string`. 👍🏽